resources/definitions/kemiq_q2_beta.def.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "kemiq_q2_beta",
+    "version": 2,
+    "name": "Kemiq Q2 Beta",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "KEMIQ",
+        "manufacturer": "KEMIQ",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "platform": "kemiq_q2.stl",
+		"has_machine_quality": true,
```

Comment:
Please use `spaces` instead `TABs`, and the same applies to the lower part and `kemiq_q2_gama.def.json`